apps/condo/domains/acquiring/utils/serverSchema/index.js

```diff
@@ -83,6 +84,20 @@ async function allPaymentsSum (context, data) {
 
 const RecurrentPaymentContext = generateServerUtils(RecurrentPaymentContextGQL)
 const RecurrentPayment = generateServerUtils(RecurrentPaymentGQL)
+
+async function paymentByLink (context, data) {
```

Comment:
Better to name mutations with verb
createPaymentByLink

Reply:
Refactored